keyboards/rocketboard_16/rules.mk

```diff
@@ -2,7 +2,18 @@
 MCU = STM32F103
 
 # Bootloader selection
-BOOTLOADER = stm32duino
+# BOOTLOADER = stm32duino
+
+# Custom loader configuration
+MCU_LDSCRIPT = STM32F103C8T6_custom
```

Comment:
```suggestion
MCU_LDSCRIPT = STM32F103x8
```

Reply:
I suppose that should be `MCU_LDSCRIPT = STM32F103xB_stm32duino_bootloader` instead?

Reply:
Yes! Was just trying to figure out what was going on, @sigprof is correct, with `MCU_LDSCRIPT = STM32F103xB_stm32duino_bootloader` it bootloads just fine. Going to do a bit more testing, but then I will commit/push these changes. Thanks!